db/migrate/20221013090105_remove_role_from_users.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveRoleFromUsers < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :users, :role, :string
+  end
+end
```

Comment:
- [ ] An error occurs in your migrations as a result of this. Are you sure you want to remove the `role` column? Please see the error below and try to fix it 👇 
![image](https://user-images.githubusercontent.com/46839548/195709414-cc91b680-0e5b-4687-818c-343e7fcbea0f.png)